portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro

```diff
@@ -893,6 +893,24 @@ definition {
 		}
 	}
 
+	macro AssertSelectedTags {
```

Comment:
Please be more specific with the assertion you are making (is element present, not visible, etc)